activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -55,12 +59,22 @@ def sql(event)
 
     def sql_color(sql)
       case sql
-        when /\s*\Ainsert/i      then GREEN
-        when /\s*\Aselect/i      then BLUE
-        when /\s*\Aupdate/i      then YELLOW
-        when /\s*\Adelete/i      then RED
-        when /transaction\s*\Z/i then CYAN
-        else MAGENTA
+        when /\A\s*rollback/mi then
```

Comment:
Thanks.  I am still used to writing code that is backwards compatible with Ruby vOLD. :P